index.bs

```diff
@@ -838,6 +838,64 @@ interface ReceiveStream {
 ReceiveStream includes IncomingStream;
 </pre>
 
+# Protocol Mappings # {#protocol-mapping}
+
+This section specifies the QUIC protocol actions expected from methods defined
+in this specification, utilizing [[WEB-TRANSPORT-HTTP3]].
```

Comment:
I think this section needs to be non-normative.

I see some value in it as an overview of typical behavior, but the normative steps are more involved and need to be provided in algorithms for bi and uni streams, similar to what is being done for datagrams in https://github.com/w3c/webtransport/pull/192#issue-572955808 which has hooks for `writeDatagrams` and `readDatagrams` that I believe @yutakahirano is working on in patches.

Once such patches are merged, this section would be competing normatively otherwise.